packages/test-runner-puppeteer/src/puppeteerLauncher.ts

```diff
@@ -1,13 +1,12 @@
 import * as puppeteer from 'puppeteer';
+import * as puppeteerCore from 'puppeteer-core';
+import { Browser, Page } from 'puppeteer-core';
```

Comment:
These imports are added as a workaround for https://github.com/puppeteer/puppeteer/issues/6904, let's wait for it to be fixed.